webrender/src/clip_scroll_tree.rs

```diff
@@ -109,10 +109,9 @@ impl ClipScrollTree {
     pub fn get_scroll_node_state(&self) -> Vec<ScrollLayerState> {
         let mut result = vec![];
         for (id, node) in self.nodes.iter() {
```

Comment:
offtopic: I think this could be neatly rewritten as `iter().flat_map(...).collect()`